bin/lando/setup.sh

```diff
@@ -46,6 +46,9 @@ wp config create \
 	--dbpass=$DB_PASS \
 	--dbhost=$DB_HOST \
 	--extra-php <<PHP
+define( 'VIP_STATSD_HOST', 'statsd' );
+define( 'VIP_STATSD_PORT', 8125 );
```

Comment:
```suggestion
define( 'VIP_STATSD_PORT', 8126 );
```

Reply:
I think these configs actually need to be defined in `configs/wp-config-defaults.php` - I tried rebuilding the environment but these didn't get included (and the other configs are defined there).

Reply:
Oh! I didn't notice that `wp-config-defaults.php` was for Lando only. Certainly, it's a better place to put them, I'll change it, thanks!